server/src/controllers/verifyUser.js

```diff
@@ -0,0 +1,17 @@
+const jwt = require('jsonwebtoken');
+require('env2')('./config.env');
+const getUser = require('../database/queries/getUser');
+
+module.exports = (req, res, next) => {
+  jwt.verify(req.cookies.token, process.env.SECRET_KEY, (err, decoded) => {
+    if (decoded !== undefined) {
+      getUser(decoded.userID).then((users) => {
+        [req.user = null] = users.rows;
+        next();
+      });
+    } else {
+      req.user = null;
```

Comment:
and i want this middleware to send the user who is not logged in: `403` immediately instead of letting each route decide what he wanna do just like the checkAdmin middleware